Resources/TuxHook2/inject.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+STEAM_PID=$(ps -C steam -o pid= | head -1)
```

Comment:
Why not `pidof steam`?

Reply:
Dunno, i'm no bash expert.

Reply:
I also forgot to add passing PID by argument...

Reply:
And other processes (srcds et al)?